src/components/Header/index.tsx

```diff
@@ -333,7 +350,9 @@ export default function Header() {
                 </Trans>
               </BalanceText>
             ) : null}
-            <Web3Status />
+            <Trace page={currentPage}>
```

Comment:
i think i'm not following why this page trace lives in the page header, vs having it live in `App.tsx`

Reply:
yeah good callout, seems reasonable to me to move it to app.tsx. done.